engine/test/bindings/python/test.py

```diff
@@ -34,6 +34,22 @@
     limit=2,
     with_distance=True
 )
+
+print(code, response)
+
+pk_to_delete = [1, 2, 3, 4]
+print("deleting pk ", pk_to_delete)
+code = epsilla.delete_by_pk(table_name="MyTable", primary_keys=pk_to_delete)
```

Comment:
Can we call this just delete? And use primary_keys and future filter condition as positional parameters to distinguish which underneath function to call?

Reply:
sure. https://github.com/epsilla-cloud/vectordb/pull/50/commits/329e4bde3fc5eaf0a00fe7b8b07a7b08a85b4d1c